README.md

```diff
@@ -137,6 +137,16 @@ These comments can contain environment variables that will be translated when po
 
 This feature can be used to post for instance a URL to the deployed application or code coverage at a successful build and why the build failed like what tests that did not pass.
 
+## Pipeline support
+
+Support for Jenkins pipelines is currently experimental. To enable it, go to *Manage Jenkins*, then *Configure System* and check for *Enable Pipeline Support*.
+
+The configuration is the same as for other projects, but it is not currently possible to customize the comments posted to the Bitbucket Server after the job completion.
+
+Stash Pull Request Builder Plugin could serve as a stepping stone to a more advanced setup using Bitbucket Branch Source Plugin. Unlike the later, this plugin supports inline Groovy scripts for pipeline configuration, which may be handy while figuring out the optimal configuration for the build. Once that configuration is established, it can be added to the sources as `Jenkinsfile`.
+
+Stash Pull Request Builder Plugin makes conversion to pipelines easier the users, as it behaves the same way as before, it just triggers a pipeline project instead of a freestyle project. This plugin works with pull requests only and relies on comments for deciding when and what to build. It would not scan the git repository for all branches and build them all. It will never build a branch before a pull request is created from it.
+
```

Comment:
@proski can you add an example usage in Pipeline? There's the snippet generator,  but it might be convinient to also have it in the docs.

Reply:
That's only needed when Jenkinsfile is used. I've added text how to use the snippet generator. The hard part for users might be to find the configuration in the snippet generator, so I mentioned that it's in "properties".

I don't feel good about writing the script in README.md because I would need to provide some values for cron, credentials, project, repository and Stash API. They all are system specific. Also, it would make pipeline support harder than it is. No special code is needed for the common case without Jenkinsfile.

Reply:
Fair enough, but I think we need to settle on a particular symbol (c.f. #73) before doing a release. Changing it later would make the previous `Jenkinsfiles` incompatible with new version I think

Reply:
I believe calls with the class name would still work after we add a symbol. I'll check it later today. We call pipeline support experimental for now, so we can rename things. I think we should eventually rename the plugin to "Bitbucket Server PR Builder", in which case we would need to limit the exposure of the term "stash" is user visible interfaces. A symbol would help with that. Stash Notifier uses two symbols. I want to avoid it. https://github.com/jenkinsci/stashnotifier-plugin/blob/release/1.x/src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java#L526

Reply:
I would still like to avoid breaking peoples Jenkinsfiles if possible. You are probably right that direct classname should still work after symbol is added though.